test/units/testsuite-85.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+# shellcheck disable=SC2016
+set -eux
+set -o pipefail
+
+if systemd-detect-virt --quiet --container; then
+    echo "Running on a container, skipping."
+    touch /skipped
+    exit 0
+fi
+
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
+
+: >/failed
+
+# Check that we're the ones to receive coredumps
+sysctl kernel.core_pattern | grep systemd-coredump
+
+mkdir -p /run/systemd/coredump.conf.d/
+printf '[Coredump]\nStorage=external' >/run/systemd/coredump.conf.d/99-external.conf
+
+ulimit -c unlimited
+systemctl start coredumping.service
+sleep 1
+
+killall --verbose -s ABRT sleep
```

Comment:
```
08:25:17 [    6.802379] testsuite-85.sh[392]: + systemctl start coredumping.service
08:25:17 [    6.840227] testsuite-85.sh[392]: + sleep 1
08:25:17 [    7.832620] testsuite-85.sh[392]: + killall --verbose -s ABRT sleep
08:25:17 [    7.841950] testsuite-85.sh[406]: sleep: no process found
```

this is not reliable - instead, do something like 'systemctl kill --signal=ABRT coredumping.service'